src/main/java/com/fizzbuzz/printers/FizzBuzzPrinter.java

```diff
@@ -0,0 +1,14 @@
+package com.fizzbuzz.printers;
+
+import com.fizzbuzz.strategy.FizzBuzzStrategy;
+
+/**
+ * Created by Bill on 17/5/2017.
+ */
+public class FizzBuzzPrinter {
+
+    public String printResult(Integer number){
+        FizzBuzzStrategy fizzBuzzStrategy = new FizzBuzzStrategy();
```

Comment:
Well my main idea was that the functionality of the FizzBuzz class is printing something therefore there is the FizzBuzzPrinter to deliver the printing result. This might defer based on the strategy you are going to use. 
Thats why i created a Strategy class which i was going to change into a interface and 3 different substrategies which are implementing it and are called FizzStrategy BuzzStrategy and FizzBuzzStrategy.